src/KeyVault/Microsoft.Azure.KeyVault.Cryptography/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.0.2-preview",
+  "version": "2.0.3-preview",
```

Comment:
Because the cryptography library package was published before adding the AssemblyInfo file. This is to publish the package with AssemplyInfo to include the right file version.